utils/static/js/pager.js

```diff
@@ -25,12 +25,15 @@ $(function() {
   var queries = {};
   $.each(document.location.search.substr(1).split('&'), function(c, q) {
```

Comment:
How about wrap the url parameter selection in a funciton or selector?
I know jQuery doesn't have the function. Implement by ourselves is a good choice?